apidoc/Titanium/UI/TextField.yml

```diff
@@ -621,6 +621,10 @@ events:
 
   - name: return
     summary: Fired when the return key is pressed on the keyboard.
+    description: |
+            Since in Landscape mode Android provides both the New Line and the Action type
+            buttons as return keys the 'button' property in the event is used to distinguish from
+            which key has the event been fired.
```

Comment:
Let's get rid of this description since we're not offering this new property yet.